README.md

```diff
@@ -76,6 +78,50 @@ pipeline:
     command: echo "hello, world, command_stage_3"
 ```
 
+HCL (all files with .hcl prefix are assumed to be of HCL format)
+```
+//The pipeline
+pipeline {
+  stage {
+    name = "command_stage_1"
+    type = "command"
+    command = "echo 'hello, world'"
+  }
+  stage {
+    name = "command_stage_2"
+    type = "command"
+    command = "echo 'hello, world, command_stage_2'"
+  }
+  stage {
+    name = "command_stage_3"
+    type = "command"
+    command = "echo 'hello, world, command_stage_3'"
+  }
+}
+```
+
+JSON
+```json
+{
+  "pipeline":{
+    "stage" :{
+      "name":"command_stage_1",
+      "type":"command",
+      "command":"echo 'hello, world'"
+    },
+    "stage":{
```

Comment:
Sorry, I haven't played around enough with HCL or Walter to give you a good recommendation here. Just noticed the JSON is invalid.